### PR TITLE
fix(frontend): Change the prompt and error message of parameter field into more readable text

### PR DESCRIPTION
**Pull request by jlyaoyuli**
Previously, KFP directly use proto type as error message of parameter field. The main purpose of this change is change the type in the text more readable, which aligns with MP's design.

The changes as follow:
```
NUMBER_INTEGER -> integer
NUMBER_DOUBLE -> double
BOOLEAN -> boolean
STRING -> string
LIST -> list
STRUCT -> map / dict
```

Before:
Screenshot 2023-02-08 at 11 36 02 AM

After:
Screenshot 2023-02-08 at 11 37 39 AM

### CODE REVIEW

File: frontend/src/components/NewRunParametersV2.tsx [hunk 1]
```diff
@@ -60,6 +60,15 @@ interface NewRunParametersProps {
   setIsValidInput?: (isValid: boolean) => void;
 }
 
+const protoMap = new Map<string, string>([
+  ['NUMBER_DOUBLE', 'double'],
+  ['NUMBER_INTEGER', 'integer'],
+  ['STRING', 'string'],
+  ['BOOLEAN', 'boolean'],
+  ['LIST', 'list'],
+  ['STRUCT', 'map / dict'],
```
Comment (zijianjoy) [inline frontend/src/components/NewRunParametersV2.tsx:69]: NIT: I am wondering what is the context of these types? Are we referring to them as Python class?

Comment (jlyaoyuli) [reply frontend/src/components/NewRunParametersV2.tsx:69]: I think we are referring to python class!

**Review by zijianjoy — COMMENTED** [review]
/lgtm
/approve

Thank you Joe!

**Pull request merged**